RetailCoder.VBE/UI/RubberduckUI.de.resx

```diff
@@ -1114,10 +1114,10 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren.</value>
   <data name="RubberduckMenu_Navigate" xml:space="preserve">
     <value>Na&amp;vigieren</value>
   </data>
-  <data name="Language_JP">
-    <value />
+  <data name="Language_JP" xml:space="preserve">
+    <value>Japanisch</value>
   </data>
-  <data name="Inspections_MultilineParameter">
-    <value />
+  <data name="Inspections_MultilineParameter" xml:space="preserve">
+    <value>Varibalen in einer Zeile angeben</value>
```

Comment:
I fare that should be "Variablen", but that's all :grin: , thanks for the cleanup :+1: